public/css/style.css

```diff
@@ -626,7 +626,7 @@ footer .links{
 
 footer .links li {
 	float: left;
-	width: 20%;
+	width: 14%;
```

Comment:
A bit crummy that we have to do this to fit links in. Might be worth reconsidering how this is done in future.